.travis.yml

```diff
@@ -33,6 +33,26 @@ matrix:
     - php: 7
       env:
         - DEPS=latest
+    - php: 7.1
+      env:
+        - DEPS=lowest
+    - php: 7.1
+      env:
+        - DEPS=locked
+        - CS_CHECK=true
```

Comment:
There is no need to run cs-check on every PHP version - please remove it here and also on PHP 7.2.